javagen/src/main/java/com/azure/autorest/model/javamodel/JavaJavadocComment.java

```diff
@@ -37,6 +38,16 @@ public final void description(String description) {
         line(processedText);
     }
 
+    public final void description(String description, boolean withStartAndEndGeneratedComment) {
```

Comment:
Personally, prefer this to be in a util class. This not exactly belong to `javamodel`.

Reply:
Yes, it does not have meaning in Java. But I did not find a util class very suitable for putting this code. Do you have recommendation. 
@srnagar would also like to hear your thoughts on this before doing the change.

Reply:
`TemplateUtil.writeJavadocDescriptionWithGenerated` seems good enough.

Reply:
Yeah, I am fine with moving this to a util method.